react-components/select/step4.md

```diff
@@ -0,0 +1,71 @@
+Another customization option, available for all variants, is custom display and data for the Select options.
+
+Let's begin with customizing the data. To change what data is passed around, and back to you on selection, you must implement a SelectOptionObject class. The only required property is a toString function, which converts whatever other data you include in the structure to a string format. This allows the component to internally compare options for filtering, and gives some limited customization for the option's display.
+
+1. Add SelectOptionObject to the list of imports from @patternfly/react-core.
```

Comment:
Same comment here as the previous step, show the user what their import statement should now look like.

Reply:
wrap SelectOptionObject in code ticks